### DISCUSSION TITLE
Linux pulse audio

### DISCUSSION THREAD

**Discussion by parabyte** [pschatzmann/arduino-audio-tools#1497] [Q&A] [answered]
I do not really wish to reinvent the wheel, is there any projects out there that will cause the esp32 to listen for pcm data from say pulse audio on linux? otherwise happy to roll my own.

**Comment by pschatzmann** [comment, marked as answer]
You can copy audo data via UDP or PCM. 
The easiest solution however is to use snapcast e.g. with my snapclient

**Comment by parabyte** [comment]
wow thank you, today's lesson, always have a good look around someones github projects! 
I was completely unaware of snap audio servers/clients until just now! thanks and VERY happy !!
